src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {

```

Comment:
It's good practice to include access modifiers for constructors to make the code's intention clear. If you intend for the constructor to be package-private (which is the default when no modifier is specified), it's still a good idea to explicitly declare it for the sake of clarity.